src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <!-- Personal info form -->
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset name="field-block">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label> Surname:
+            <input type="text" name="surname"autocomplete="off">
+          </label>
+        </div>
+        <div class="form-field">
+          <label> Name:
+            <input type="text" name="name" autocomplete="off">
+          </label>
+        </div>
+        <div class="form-field">
+          <label> How old are You?
+            <input type="number"
+            name="counter"
+            value="12"
+            min="1"
+            max="100"
+            >
```

Comment:
if more than 2-3 properties, move each prop to a new line
fix formatting everywhere

```suggestion
            <input
              type="number"
              name="counter"
              value="12"
              min="1"
              max="100"
            >
```